README.md

```diff
@@ -100,3 +103,5 @@ I would like to thank God to help me finish this project
 This project is [MIT](./LICENSE) licensed.
 
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```

Comment:
- [ ] [**OPTIONAL**] This note is only instructions for you to set up the license. I think you should remove it to keep your README looking professional.